### PR TITLE
fix: add width, height and crossOrigin props on Image type declaration file

### PR DESCRIPTION
**Pull request by alvessteve**
## Summary

According this Issue [`#36182`](https://github.com/facebook/react-native/issues/36182), the typescript compiler doesn't know the `width`, `height` and `crossOrigin` props for the `Image` component although they are taken in account in the code, [docs](https://reactnative.dev/docs/image#crossorigin) and the flow types.

## Changelog

[GENERAL] [FIXED] - Fix missing `height`, `width`, `crossOrigin` props on Typescript Image.d.ts file

### CODE REVIEW

**Comment by carlosalmonte04** [comment]
`Image.d.ts` is also missing `objectFit`, `referrerPolicy`, `src`, `srcSet`, `tintColor` which are found in the docs. I'd say either submit a PR for all of the missing props or just submit a PR targeting `height` and `width` only. Otherwise crossOrigin is just random for this PR.

**Comment by rshest** [comment]
@alvessteve Looks ok to me, but please do take into account the @carlosalmonte04 comment.

**Review by rshest — APPROVED** [review]

**Comment by carlosalmonte04** [comment]
TL;DR: It's more organized having PR's be associated to a specific task.

I guess it's not a big deal around here. `crossOrigin` deviated from the issue and it didn't fix the other problem fully either, which was adding all the props from the docs.

**Comment by alvessteve** [comment]
thank you for the comments @carlosalmonte04 and @rshest ! As it was no big deal here, I put them together but next time I will split it.

**Comment by carlosalmonte04** [comment]
Thank you. Please do. Otherwise it brings down repo morale and reputation.
